### PR TITLE
Create a CONTRIBUTING.md file and update README.md

### PR DESCRIPTION
**Pull request by amoweolubusayo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +36,7 @@ You can be as creative as you want with the tagging system. For example, you cou
 How does it work?
 -----------------
 
-In short, the app works this way - Bookmark the relevant folders, add tags to the items, filter the items using the added tags.
+In short, the app works this way - Bookmark the relevant folders, view contents of folder, add tags to the items and filter the items using the added tags.
```
Comment (kirillt) [inline README.md:39]: It's also worth mentioning that one of key features is "aggregated mode". That's what happening when you just press "navigate data" button in the bottom bar. It takes resources from all folders and presents the single resource space to the user, allowing him to navigate this space using power of tags. Although we are still working on performance and UX of course, so selecting individual folders is faster sometimes.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+🎈 You made it here! This is a great step in helping to contribute to ARK Navigator. 
+
+To get started, you can start off here [issues](https://github.com/ARK-Builders/ARK-Navigator/issues) with those tagged "good first issue" 
+
+The projects embeds ktlint. You can check it out in GitHub Actions workflow.
+
+## Creating a Pull Request / Pull Request Rules
+
+Before a PR can be merged, you would have to fix all code style errors.
```
Comment (kirillt) [inline CONTRIBUTING.md:12]: "We also care a lot about our software quality, that's why we are conducting code review before merging. Right now, the team isn't that big, so please be patient."

Comment (amoweolubusayo) [reply CONTRIBUTING.md:12]: Thanks for this, it has been updated

File: README.md (hunk 1 above)
Comment (amoweolubusayo) [reply README.md:38]: Thanks for this, it has been updated

**Review by kirillt — APPROVED** [review]
Thanks!

**Pull request merged**